src/services/sports/index.jsx

```
@@ -0,0 +1,400 @@
+        }
+        isOpen={isOpen}
+        onClose={onClose}
+        size="lg"
+      >
+        {/* Image Previewer */}
+        <div className="previewer">
+          <img
+            style={{ imageRendering: "-webkit-optimize-contrast" }}
+            src={image?.image}
+            className="w-full h-full object-contain"
+          />
```

Biome (1.9.4)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)